File: AppHogar/src/main/java/co/edu/poli/edu/AppHogar/entity/FacturaServicio.java
package co.edu.poli.edu.AppHogar.entity;

import java.time.LocalDate;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class FacturaServicio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private double monto;
    private LocalDate fechaVencimiento;
    private boolean pagado;

    @ManyToOne
    private Proveedor proveedor;
}

File: AppHogarFront/Models/FacturaServicio.cs
using System;

namespace AppHogarFront.Models
{
    public class FacturaServicio
    {
        public long Id { get; set; }
        public double Monto { get; set; }
        public DateTime FechaVencimiento { get; set; }
        public bool Pagado { get; set; }
        public Proveedor Proveedor { get; set; }
    }
}
